src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.dao;
+
+import java.util.List;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+    Order add(Order order);
+
+    Order getByUser(User user);
+
+    List<Order> getAll();
+
+    List<Order> getOrdersHistory(User user);
+
+    void update(Order order);
```

Comment:
```suggestion
    Order add(Order order);
    
    List<Order> getOrdersHistory(User user);
```

we don't need these methods

Reply:
no means no)